pipeline/transform.py

```diff
@@ -0,0 +1,17 @@
+"""Transforms the plant data to make it ready for loading"""
+import pandas as pd
+
+
+def clean_plant_data(plant_df: pd.DataFrame) -> pd.DataFrame:
+    """Setting the types for dataframe columns and removing NaN values"""
+
+    plant_df['recording_at'] = pd.to_datetime(plant_df['recording_at'])
+    plant_df['last_watered'] = pd.to_datetime(plant_df['last_watered'])
+    plant_df['soil_moisture'] = pd.to_numeric(plant_df['soil_moisture'], errors='COERCE')
+    plant_df['temperature'] = pd.to_numeric(plant_df['temperature'])
+    plant_df['plant_id'] = pd.to_numeric(plant_df['plant_id'])
```

Comment:
Could you add `errors='coerce'` to all of them? Otherwise rather than changing an invalid value to NAN it will raise an exception